fuse/data/utils/samplers.py

```diff
@@ -44,7 +44,8 @@ def __init__(
     ) -> None:
         """
         :param dataset: dataset used to extract the balanced class from each sample
-        :param balanced_class_name:  the name of balanced class to extract from dataset
+        :param balanced_class_name:  the name of balanced class to extract from dataset.
+                                     Set to none if you just want to randomly sample from entire dataset.
```

Comment:
Nice :)

I suggest that we will have a unit-test for this option (in a future PR).